lambda-ssm-sls/README.md

```diff
@@ -0,0 +1,66 @@
+# AWS Lambda GET and PUT to Amazon SSM Parameter Store Using the Serverless Framework
```

Comment:
Missing example-pattern.json on this pattern, if you could add that would be great :)